site/source/pages/Simulateurs/NextSteps.tsx

```diff
@@ -86,4 +86,16 @@ const guidesUrssaf = [
 		associatedRule: 'dirigeant',
 		title: 'Guide Urssaf pour les indépendants',
 	},
+	{
+		url: 'https://www.urssaf.fr/portail/home/employeur/employer-du-personnel/nouvel-employeur.html',
+		title: "Nouvel employeur : l'Urssaf vous accompagne",
+		description:
+			'Vous employez votre premier salarié et vous vous posez des questions sur les démarches à effectuer ? Ce guide est fait pour vous.',
+		associatedRule: {
```

Comment:
Une alternative : considérer que l'affichage de ces liens pdf relève de l'éditorial au même titre des textes qu'on affiche sous les simulateurs, et faire un “opt-in” uniquement sur les pages sur lesquelles on veut afficher le lien `<NextStepPdfLink />`. Ça évite d'avoir ce guide d'embauche sur la page chômage partiel par exemple.

(évidemment pas bloquant pour merger, vu que c'est un plus gros changement que le seul ajout d'un pdf avec la logique existante)

Reply:
Oui, c'est clair. Je ne suis pas trop à l'aise avec cette version qui éclate le contenu des liens utiles (`NextStep`) à travers plusieurs endroit de la base de code (`metada-scr`, NextSteps.tsx`, ...). Je préfèrerai que ce soit directement dans les pages d'explications, avec une factorisation si besoin.